docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1346,6 +1346,66 @@ privacy-url="http://127.0.0.1:8888"
 
 URL on which the [Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is running.
 
+### pruning-block-confirmations
+
+!!! caution 
+    Do not use pruning in Hyperledger Besu v1.4.0. Pruning has a
+    [known bug](https://github.com/hyperledger/besu/blob/master/CHANGELOG.md#known-issues).
+    
+    If using fast sync in v1.4.0, explicitly disable pruning using `--pruning-enabled=false`.
+
+```bash tab="Syntax"
+--pruning-block-confirmations=<INTEGER>
+```
+
+```bash tab="Command Line"
+--pruning-block-confirmations=5
+```
+
+```bash tab="Environment Variable"
+BESU_PRUNING_BLOCK_CONFIRMATIONS=5
+```
+
+```bash tab="Configuration File"
+pruning-block-confirmations=5
+```
+
+The minimum number of confirmations on a block before marking begins. The default is 10.
```

Comment:
what is marking?

Reply:
Spoke with Abdel about this and have updated the description.